packages/apps/plugins/plugin-markdown/src/types.ts

```diff
@@ -69,3 +71,17 @@ export type MarkdownPluginProvides = SurfaceProvides &
   SettingsProvides<MarkdownSettingsProps> &
   TranslationsProvides &
   StackProvides;
+
+export const DocumentSchema = S.struct({
+  title: S.optional(S.string),
+  content: S.optional(S.string),
+  comments: S.optional(
+    S.struct({
+      // TODO(wittjosiah): Add thread schema.
+      // thread: S.optional(ThreadSchema),
+      cursor: S.optional(S.string),
+    }),
+  ),
+}).pipe(E.echoObject('dxos.sdk.client.Properties', '0.1.0'));
```

Comment:
as long as the typename exactly matches the ones generated from protobuf, the old data should be usable as well

Reply:
we can think about migrating those names later